frontend/src/component/Login/SignIn.jsx

```diff
@@ -38,13 +38,20 @@ const SignIn = props => {
             </Link>
           </li>
           {props.system.config.idp_list.map((idp) =>
-            <li key={idp}>
+            <li key={idp.identifier}>
               <a
-                className='loginpage__main__form__forgot_password'
-                href={'/saml/sso?target=' + encodeURIComponent(idp)}
+                className='loginpage__main__form__saml'
```

Comment:
I do not understand why you replace `forgot password` with `saml` in a generic component. Do I understand wrong?

Reply:
After investigation, I feel like the diff is not the real diff (you diff with some already included SAML related code, isn't it?)

Reply:
the `forgot_password` class was used because it initially looked OK for this usage. In this PR since I changed the look of the IdP list I created specific classes for SAML IdP list

Reply:
As said earlier, the diff is with #6232, since it is the current target. So the diff that are shown are related with already included SAML code